### ISSUE TITLE
TableViewController not being added to child view controllers

### ISSUE THREAD

**Issue by markst** [mikeRozen/MRDropDown#3]
See: https://developer.apple.com/library/archive/featuredarticles/ViewControllerPGforiPhoneOS/ImplementingaContainerViewController.html

**Comment by mikeRozen** [comment]
Yes, Currently I don't add TableViewController as child controller I only add tableView as a subview.
....
